sycl/test-e2e/Plugin/interop-level-zero-get-native-mem.cpp

```diff
@@ -2,8 +2,8 @@
 // L0 plugin incorrectly reports memory leaks because it doesn't take into
 // account direct calls to L0 API.
 // UNSUPPORTED: ze_debug-1,ze_debug4
-// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %level_zero_options %s -o %t.out
-// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: %{build} %level_zero_options -o %t.out
+// RUN: %{run} %t.out
```

Comment:
This test probably originally needed something to ensure it was only exercised against L0.  Again a `//REQUIRES: level_zero` .  Let me know if I'm misunderstanding.